components/Footer.js

```diff
@@ -1,8 +1,89 @@
+import Link from 'next/link';
+
 export default function Footer() {
 	const year = new Date().getFullYear();
 	return (
-		<footer>
-			<p className='footer--footer'>&copy; Erick Askim {year}</p>
-		</footer>
+		<div className='footer--wrapper'>
+			<div className='footer--menu'>
+				<h2 className='footer--title'>Erick Askim</h2>
+				<ul className='footer--menu__ul'>
+					<li>
+						<Link href='/'>
+							<p>Home</p>
+						</Link>
+					</li>
+					<p className='footer--social__line'>─</p>
+					<li>
+						<Link href='/about'>
+							<p>About</p>
+						</Link>
+					</li>
+					<p className='footer--social__line'>─</p>
+					<li>
+						<Link href='/'>
+							<p>Services</p>
+						</Link>
+					</li>
+					<p className='footer--social__line'>─</p>
+
+					<li>
+						<Link href='/'>
+							<p>Portfolio</p>
+						</Link>
+					</li>
+					<p className='footer--social__line'>─</p>
+					<li>
+						<Link href='/blog'>
+							<p>Blog</p>
+						</Link>
+					</li>
+					<p className='footer--social__line'>─</p>
+					<li>
+						<Link href='/contact'>
+							<p>Contact</p>
+						</Link>
+					</li>
+				</ul>
+			</div>
+			<div className='footer--info'>
+				<div className='footer--address'>
+					<address className='footer--address-wrap'>
+						<h3 className='footer--h3'>Address:</h3>
```

Comment:
Headings are not valid children of an address. [See here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)